modules/develop/pages/config-topics.adoc

```diff
@@ -18,15 +18,15 @@ rpk topic create xyz
 
 This command creates a topic named `xyz` with one partition and one replica, since these are the default values set in the cluster configuration file.
```

Comment:
```suggestion
This command creates a topic named `xyz` with one partition and one replica, because these are the default values set in the cluster configuration file.
```

Reply:
Or...., as these are the default values set in the cluster...

stickler for because = causal, since=passage of time